StandardRecord/SRTrackSegment.cxx

```diff
@@ -0,0 +1,29 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRTrackSegment.cxx
+// \author  jpaley@fnal.gov
+// \brief   
+////////////////////////////////////////////////////////////////////////
+#include "StandardRecord/SRTrackSegment.h"
+
+namespace caf
+{
+  SRTrackSegment::SRTrackSegment() 
+  {
```

Comment:
Add some NAN values so can just put default as 
`vtx.SetXYZ(0,0,0)`

```
vtx(std::numeric_limits<float>::signaling_NaN(),
        std::numeric_limits<float>::signaling_NaN(),
        std::numeric_limits<float>::signaling_NaN())
```

Reply:
Done.  Also set Mom elements to NaNs.